server/ot_rpc.go

```diff
@@ -83,57 +83,57 @@ func runTranscoder(n *core.LivepeerNode, orchAddr string, capacity int) error {
 			}
 			return err
 		}
-		profiles, err := common.TxDataToVideoProfile(hex.EncodeToString(notify.Profiles))
-		if err != nil {
-			glog.Info("Unable to deserialize profiles ", err)
-		}
+		go runTranscode(n, orchAddr, httpc, notify)
+	}
+}
+
+func runTranscode(n *core.LivepeerNode, orchAddr string, httpc *http.Client, notify *net.NotifySegment) {
```

Comment:
For code splits that result in a large amount of re-indentation such as this, I often find it helpful to insert a dummy set of braces to maintain the original indentation. This allows reviewers to clearly see what has changed in the commit. Afterwards, a cosmetic patch can be applied that only re-idents code without functional changes.

So this would become something like:

```
func runTranscode(n *core.LivepeerNode, orchAddr string, httpc *http.Client, notify *net.NotifySegment) {
    { // dummy indentation
        ... original code here ...
    }
}
```

Reply:
Interesting! That certainly does make it easier for the reviewer. Thanks for the tip!